test/verify/check-storage-luks

```diff
@@ -169,192 +170,179 @@ class TestStorage(StorageCase):
-        b.wait_in_text("#dialog", "A new key has been securely retrieved from the server.")
+        self.dialog_wait_apply_enabled()
+        self.dialog_wait_val("tang_url", "127.0.0.1")
+        self.dialog_set_val("tang_url", "http://127.0.0.1/")
         self.dialog_apply()
-        self.dialog_wait_close()
-
-        # Remove obsolete key, disk should still unlock
-
-        m.execute("rm /var/db/tang/.enc1.jwk")
-        m.execute("systemctl restart tangd-update")
-        if can_unlock:
-            self.content_head_action(1, "Lock")
-            b.wait_not_in_text("#detail-content", "ext4 File System")
-            self.content_head_action(1, "Unlock")
-            self.content_row_wait_in_col(2, 1, "ext4 File System")
-
-        # Rotate both the encryption key and the signing key on the server, and rebind
-
-        m.execute("mv /var/db/tang/enc2.jwk /var/db/tang/.enc2.jwk")
-        m.execute("jose jwk gen -i '{\"alg\":\"ECMR\"}' >/var/db/tang/enc3.jwk")
-        m.execute("mv /var/db/tang/sig1.jwk /var/db/tang/.sig1.jwk")
-        m.execute("jose jwk gen -i '{\"alg\":\"ES512\"}' >/var/db/tang/sig2.jwk")
-        m.execute("systemctl reset-failed tangd-update; systemctl restart tangd-update")
-
-        self.content_tab_action(1, 1, "Check")
-        self.dialog_wait_open()
-        b.wait_in_text("#dialog", "This network key is obsolete.")
-        b.wait_in_text("#dialog", "The output should match this text")
-        b.wait_in_text("#dialog", m.execute("jose jwk thp -i /var/db/tang/sig2.jwk").strip())
+        b.wait_in_text("#dialog", "Make sure the key hash from the Tang server matches:")
+        b.wait_in_text("#dialog", m.execute("jose jwk thp -i /var/db/tang/sig1.jwk").strip())
         self.dialog_apply()
         self.dialog_wait_close()
-
-        # Remove the key on the server
-
-        m.execute("rm /var/db/tang/enc3.jwk")
-        m.execute("systemctl reset-failed tangd-update; systemctl restart tangd-update")
-        self.content_tab_action(1, 1, "Check")
-        self.dialog_wait_open()
-        b.wait_in_text("#dialog", "This network key is not recognized anymore by the server.")
-        self.dialog_cancel()
-        self.dialog_wait_close()
+        b.wait_in_text(panel + ".table tr:nth-child(2)", "http://127.0.0.1/")
 
         # Remove key on client
-
-        b.wait_in_text(self.content_tab_info_row(1, 1, "Network keys"), "127.0.0.1")
-        self.content_tab_action(1, 1, "Remove")
+        #
+        b.click(panel + ".table tr:nth-child(2) .fa-minus")
         self.confirm()
-        b.wait_not_in_text(self.content_tab_info_row(1, 1, "Network keys"), "127.0.0.1")
+        b.wait_not_present(panel + ".table tr:nth-child(2)")
 
-    def testClevisHttp(self):
+    def testSlots(self):
+        self.allow_journal_messages("Device is not initialized.*")
         m = self.machine
         b = self.browser
 
-        if m.execute("which clevis || true") == "":
-            self.skipTest("No clevis/tang here")
```

Comment:
Doesn't this need some moral equivalent for OSes where clevis/tang don't exist? Or do all of our images have it now?

Reply:
Yes, indeed, and the tests already show this.  I think the plan was to replace this code with a @skipImage decorator, but I first wanted to see where it actually fails...